### PR TITLE
Test comment

### PR DESCRIPTION
**Pull request by daniel-enqz**
Hey guys can someone help me with this review, I added a new line in routes, thanks.

### CODE REVIEW

**Comment by asterixf** [comment]
looks good to me!

**Review by ba0404hx — APPROVED** [review]

**Comment by BintAmun** [comment]
Great!

File: config/routes.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   root to: "pages#home"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  # Hello pana
   # Defines the root path route ("/")
```
Comment (daniel-enqz) [inline config/routes.rb:5]: This looks very bad, what the hell is pana

Comment (ba0404hx) [reply config/routes.rb:5]: I think pana is okay

Comment (asterixf) [reply config/routes.rb:5]: yeah! is this code?

Comment (BintAmun) [reply config/routes.rb:5]: Google translate

**Pull request merged**